python/enc-config.py

```diff
@@ -36,18 +38,56 @@ def main():
     elif args.transfer:
         enc_path = paths.transfer_config
 
-    # check if a secret is provided
-    if not args.secret:
-        sys.exit("Error: secret not provided")
-
     # check if a json path is provided
     if not args.json:
         dec_path = enc_path
     else:
         dec_path = args.json
 
+    # check if a secret is provided
+    if args.secret:
+        phrase = args.secret
+
+    else:
+        from getpass import getpass
+        from base64 import b64encode
+
+        if args.mint:
+            address = input("Minter Address: ")
+            l2_pk = getpass("Layer 2 Private Key: ")
+            phrase = b64encode(getpass("Passphrase: ").encode("utf-8"))
+            royalty = input("Default Royalty (0-10): ")
+            nft_type = input("NFT Type [0 = ERC1155, 1 = ERC721]: ")
+            token = input("Fee Token [0 = ETH, 1 = LRC]: ")
+
+            config_json = {
+                "minter": address,
+                "private_key": l2_pk,
+                "royalty_percentage": royalty,
+                "nft_type": nft_type,
+                "fee_token": token
+            }
+
+        if args.transfer:
+            address = input("From Address: ")
+            l1_pk = getpass("Layer 1 Private Key: ")
+            l2_pk = getpass("Layer 2 Private Key: ")
+            phrase = b64encode(getpass("Passphrase: ").encode("utf-8"))
+            token = input("Fee Token [0 = ETH, 1 = LRC]: ")
+
+            config_json = {
+                "sender": address,
+                "private_key": l2_pk,
+                "private_key_mm": l1_pk,
+                "fee_token": token
```

Comment:
@Montspy will review to make sure the value types are correct (e.g. not strings where python expects ints)

Reply:
Type conversions added in [4912520](https://github.com/sk33z3r/LooPyGen/pull/87/commits/491252082c91a1c59f9513cbc76741f219c2f205)